stable/tomcat/values.yaml

```diff
@@ -35,6 +37,24 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+env: []
+  # - name: env
+  #   value: test
+
+extraVolumes: |
+  # - name: extra
+  #   emptyDir: {}
+
+extraVolumeMounts: |
+  # - name: extra
+  #   mountPath: /usr/local/tomcat/webapps/app
+  #   readOnly: true
+
+extraInitContainers: |
```

Comment:
this should be an empty array like `env: []` above it.

Reply:
Thanks. Addressed.